### ISSUE TITLE
Is this project up-to-date? Does the author still use this in production?

### ISSUE THREAD

**Issue by stuser81** [Gabriel439/haskell-nix#93]
Does the author @Gabriel439 still use this project in production?

If not, what have been some major modifications? Maybe overlays are one such change. What does the author think about overlays vs. overrides?

**Comment by Gabriel439** [comment]
Yeah, we still use something similar to this.  Like you mentioned, some things are out of date and we do indeed use overlays now, because overlays are composable
